env.py

```diff
@@ -159,15 +161,16 @@ def get_env_vars(
 
     organization = os.getenv("ORGANIZATION")
     repositories_str = os.getenv("REPOSITORY")
+    team_name = os.getenv("TEAM_NAME")
     # Either organization or repository must be set
     if not organization and not repositories_str:
         raise ValueError(
             "ORGANIZATION and REPOSITORY environment variables were not set. Please set one"
         )
-
-    if repositories_str and repositories_str.find("/") == 0:
+    # Team name and repository are mutually exclusive
+    if repositories_str and team_name:
         raise ValueError(
-            "REPOSITORY environment variable was not set correctly. Please set it to a comma separated list of repositories in the format org/repo"
+            "TEAM_NAME environment variable cannot be used with ORGANIZATION or REPOSITORY"
```

Comment:
I thought organization was required when using `TEAM_NAME`? 

Also maybe add check if TEAM-NAME is used then ORGANIZATION is also set?

Reply:
Good catch! I totally got that messed up on the wording. Fixing...

I originally had in here to check if TEAM-NAME is used then ORGANIZATION is also set but the code was unreachable because we already check for ORGANIZATION or REPOSITORY, and then for TEAM_NAME and not REPOSITORY, so by deduction we've confirmed ORGANIZATION is present.

Reply:
That's a good point re: unreachable code. I'm a fan of explicit over implicit. Not a blocker though.